tools/upgrade-deps/src/index.js

```diff
@@ -121,13 +117,13 @@ async function upgradeYarnProject(program) {
 	const changed = await npm.checkPackageJson(`${CWD}/package.json`, opts);
 	if (!opts.dry) {
 		if (!opts.scope && !opts.package && !opts.startsWith) {
-			commands.unshift('yarn upgrade');
+			commands.unshift('yarn upgrade --ignore-scripts');
```

Comment:
Maybe it would be nice to have an option to activate it or not form the command line WDYT? because for CI I understand the point but when running locally